curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/655b4dad1d38ff7cdd65cbfe.md

```diff
@@ -551,38 +551,73 @@ const shuffleButton = document.getElementById("shuffle");
 const allSongs = [
```

Comment:
We also need to update the final seed code because that is where the preview comes from

Right now, the preview still shows the original artwork and songs
<img width="662" alt="Screenshot 2023-12-17 at 9 59 55 AM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/67210629/39ff48c8-1884-4ee6-8c24-60f557de2fa5">

The lines you need to update start on lines 1321